src/dotnet/CarbonAware/Model/Location.cs

```diff
@@ -19,16 +23,18 @@ public class Location
     /// </summary>
     public decimal? Longitude { get; set; }
 
+
     /// <summary>
-    /// Gets or sets the type of location this location object represents.
+    /// Gets or sets the cloud provider name to use.  When loation type is GeoPosition, this value should be null..
     /// </summary>
-    public LocationType LocationType { get; set; } = LocationType.NotProvided;
+    public CloudProvider? ProviderName { get; set; }
```

Comment:
Should be the variable be renamed to something else? rather than `ProviderName`?

Reply:
Suggestions?  Should it just match the enum and be `CloudProvider` ?

```
Location.LocationType // CloudProvider
Location.ProviderName // Azure
Location.RegionName // eastus
```
VS

```
Location.LocationType // CloudProvider
Location.CloudProvider // Azure
Location.RegionName // eastus
```